internal/version/version.go

```diff
@@ -248,3 +250,27 @@ func (i *Info) JSONString() (string, error) {
 	}
 	return string(b), nil
 }
+
+// GetLatestReleaseVersion fetches the latest github release
+// version from cri-o.
+func GetLatestReleaseVersion(token string) (string, error) {
+	ctx := context.Background()
+	client := github.NewTokenClient(ctx, token)
+	tags, _, err := client.Repositories.ListTags(ctx, "cri-o", "cri-o", nil)
+	if err != nil {
+		return "", err
+	}
+	rng, err := semver.ParseRange(fmt.Sprintf(">= %s", Version))
```

Comment:
can there be some comments about this and the next couple of steps? it's not immediately obvious what's happening here